client/my-sites/subscribers/components/grow-your-audience/styles.scss

```
@@ -0,0 +1,84 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@automattic/typography/styles/variables";
+@import "@wordpress/base-styles/breakpoints";
+
+.grow-your-audience {
+	color: $studio-gray-100;
+	font-family: "SF Pro Text", $sans !important;
+	padding: 64px 0;
+
+	.grow-your-audience__title {
+		font-weight: 500;
+		font-size: $font-title-small;
+		line-height: rem(26px);
+		margin-bottom: 16px;
+	}
+
+	.grow-your-audience__cards {
+		display: flex;
+		gap: 32px;
+	}
+
+	.grow-your-audience__card {
+		box-shadow: none;
+		border: 1px solid $studio-gray-5;
+		border-radius: 4px;
+		flex-basis: 33.3%;
+
+		.components-card__body {
+			padding: 16px 24px;
+		}
+	}
+
+	.grow-your-audience__card-title {
+		color: $studio-gray-100;
+		display: flex;
+		align-items: center;
+		gap: 4px;
+		font-weight: 500;
+		font-size: $font-body-small;
+		line-height: rem(20px);
+		margin-bottom: 8px;
+	}
+
+	.grow-your-audience__card-icon {
+		flex-basis: 20px;
+	}
+
+	.grow-your-audience__card-text {
+		color: $studio-gray-60;
+		font-size: $font-body-extra-small;
+		line-height: rem(20px);
+	}
+
+	.grow-your-audience__card-link {
+		display: block;
+		font-weight: 500;
+		font-size: $font-body-small;
+		margin-top: 16px;
+	}
+}
+
+@media screen and (max-width: $break-xlarge) {
+	body:not(.is-sidebar-collapsed) {
+		.grow-your-audience__cards {
+			flex-direction: column;
+			gap: 16px !important;
+		}
+
+		.grow-your-audience__card {
+			flex-basis: 100%;
+		}
+	}
+}
+
+@media ( max-width: $break-medium ) {
```

Extra space on the brackets' inside:

```suggestion
@media (max-width: $break-medium) {
```